src/visitable.cpp

```diff
@@ -743,27 +758,27 @@ long visitable<inventory>::charges_of( const std::string &what, int limit ) cons
 }
 
 template <>
-long visitable<Character>::charges_of( const std::string &what, int limit ) const
+long visitable<Character>::charges_of( const std::string &what, long limit ) const
 {
     auto self = static_cast<const Character *>( this );
     auto p = dynamic_cast<const player *>( self );
 
     if( what == "toolset") {
         if( p && p->has_active_bionic( "bio_tools" ) ) {
-            return std::min( p->power_level, limit );
+            return std::min( (long)p->power_level, limit );
```

Comment:
> Please open another issue if you want to push static_cast as a coding style change.

Easier to fix this instance here especially if you are already intending to push another commit for that formatting?